settings

```diff
@@ -30,4 +30,4 @@ export CLIENT_VERSION="0.8-SNAPSHOT"
 # Name of the release package
 export PACKAGE_NAME="@kubernetes/node-client"
 
-export OPENAPI_GENERATOR_COMMIT=5555137b7926de2f501e091718ff72429b808220
+export OPENAPI_GENERATOR_COMMIT=ba367e60e85392a7fd58cdb374c35c3aaaece872
```

Comment:
There is already a way newer commit, should we bother updating this in this PR again?

Reply:
oh do you mean bump all the way to the newest? i was just trying to advance enough to get the media type changes, but we can hop further ahead i can update it

Reply:
Yes. At least if this doesn't introduce other problems.
Otherwise, We should merge it in the current state in my opinion.

Reply:
agreed i've updated it to the newest hash now